BuildAddOn.py

```diff
@@ -45,33 +50,42 @@ def PrepareParameters (args):
     configData = json.load (configFile)
     addOnName = configData['addOnName']
     acVersionList = None
-    languageList = None
 
     if args.acVersion:
         acVersionList = args.acVersion
     else:
         acVersionList = []
-        for version in configData['devKitLinks'][platformName]:
+        for version in devKitData[platformName]:
             acVersionList.append (version)
 
     # Get needed language codes
-    if args.release:
-        configLangUpper = [lang.upper () for lang in configData['languages']]
-        languageList = ['ALL']
-        if args.language is not None:
-            languageList = [lang.upper () for lang in args.language]
-
-        if 'ALL' in languageList:
-            languageList = configLangUpper
-        else:
-            for lang in languageList:
-                if lang not in configLangUpper:
-                    raise Exception ('Language not supported!')
+    languageList = [].append (configData['defaultLanguage'])

```

Comment:
More easy to read way:
```py
languageList = [configData['defaultLanguage']]
```

Reply:
Also, the `.upper ()` call seems missing here.

Reply:
Fixed